frame/dvm/src/lib.rs

```diff
@@ -420,6 +436,27 @@ impl<T: Config> Module<T> {
 		.into())
 	}
 
+	pub fn do_call(contract: H160, input: Vec<u8>) -> Result<Vec<u8>, DispatchError> {
```

Comment:
Why not use darwinia_evm runner call directly?

Reply:
almost all the contract methods the issuing pallet used comes from the `dvm pallet`.
here I want to keep the call graph simple.
but I'm not sure whether it's suitable to use darwinia_evm directly or to use dvm.
If we use evm directly here, we do not need to add `do_call` function in dvm.

Reply:
Please try it! No need to add this `do_all`.

```rs
pub trait dvm::Config: frame_system::Config<Hash = H256> + pallet_timestamp::Config + darwinia_evm::Config
```

Reply:
Sorry, I've rethought it again. Using darwinia-evm's runner call directly also requires abstraction work and is not yet as easy to understand as it is now. Let's keep the existing implementation and ignore my previous comments.